src/EnergyPlus/Furnaces.hh

```diff
@@ -442,7 +445,7 @@ namespace Furnaces {
 	// Functions
 
 	void
-	clear_state();
+		clear_state();
 
 	void
 	SimFurnace(
```

Comment:
Reviewed all the way through Furnaces. Things look relatively similar to existing code.